src/metrics-poller.js

```diff
@@ -0,0 +1,36 @@
+import { fetchFeatureMetrics } from './store/feature-metrics-actions';
+
+class MetricsPoller {
+    constructor(store) {
+        this.store = store;
+        this.timer = undefined;
+        this.running = false;
+
+        this.start();
+    }
+
+    start() {
+        const store = this.store;
+        store.subscribe(() => {
+            const features = store.getState().features.toJS();
+            if (!this.running && features.length > 0) {
+                this.running = true;
+                this.fetchMetrics();
+                this.timer = setInterval(this.fetchMetrics.bind(this), 5000);
+            }
+        });
+    }
+
+    fetchMetrics() {
+        this.store.dispatch(fetchFeatureMetrics());
+    }
+
+    destroy() {
```

Comment:
is this ever called? Is it a problem if it's not?

Reply:
it is via tests. It should not be a problem. Unleash will poll metrics as long as unleash is running and there are active toggles.